src/components/UI/Post.js

```diff
@@ -0,0 +1,7 @@
+import classes from './Post.module.css';
+
+function Post(props) {
```

Comment:
Can use the syntax of `{ children }` instead of `props` to make it clear what is being passed into the component and it saves having to use `props.` later on in the code.

Reply:
If you don't need a prop child in the current component but need it further down the tree should this also be in the children?

Reply:
Yes, they would also need to be included, in the same way you would have to pass them into the parent object